djtriggers/tests/test_models.py

```diff
@@ -67,6 +69,64 @@ def test_handle_execution_failure_tries_exceeded(self, mock_logger):
                                                                              try_count=original_tries+1)
         mock_logger.assert_called_with(trigger, message, level=WARNING)
 
+    @override_settings(DJTRIGGERS_TRIES_BEFORE_WARNING=3)
+    @override_settings(DJTRIGGERS_TRIES_BEFORE_ERROR=5)
+    def test_handle_execution_failure_tries_limit(self):
+        """
+        Check behavior when trigger reaches retries limit and ERROR is raised
```

Comment:
> Check behavior

Which behavior do you expect? Almost every test checks for certain behavior given certain inputs, so I'd try to be a bit more specific here. If it's hard to describe in one sentence, the test might be trying to do too much at once. A splitup might be useful in that case.

As far as I can tell, you check for two types of behavior:
- Logging calls using specific messages at a certain logging level should be done in certain circumstances
- When a trigger in a certain state (below/above retry limit) is executed, certain fields should/shouldn't have certain values after execution

My gut says that splitting might result in more readable code and more specific tests here, but I'm only looking in from the outside.

Reply:
Splitting will cause two identical tests for two distinct asserts - why not keep them in the same?

Reply:
docstring updated

Reply:
I don't know, I vaguely recall @KMahoney or @RoelandMatthijssens talking about a "one test, one assert" philosophy - which has its own set of pros and cons and I wouldn't want to apply to its extreme - and I was thinking of the benefits it might bring to separate the test's concerns a bit more. But fine by me if you think this is better.